src/framework/global/api/interactiveapi.cpp

```diff
@@ -25,16 +25,31 @@
 
 using namespace muse::api;
 
+/** APIDOC namespace: interactive
+ * User interaction - informational messages, error messages, questions and other dialogs.
+ * @namespace
+ */
+
 InteractiveApi::InteractiveApi(IApiEngine* e)
     : ApiObject(e)
 {
 }
 
+/** APIDOC method: info(title, text)
+ * Show information message
+ * @param {String} title Title
+ * @param {String} text Message
+ */
+
 void InteractiveApi::info(const QString& title, const QString& text)
```

Comment:
Could we parse some of this from the method signature, perhaps?
```C++
/** APIDOC method
 * Show really, really long
 * information message
 */
void InteractiveApi::info(
    const QString& title, /// @param Title
    const QString& text,  /// @param Message
)
{
     interactive()->info(title.toStdString(), text.toStdString());
}
```

Reply:
For this approach, we will need a type map like this https://github.com/musescore/MuseScore/blob/4ba6bf9d403a303988fa3163813e4434603b3c7a/src/framework/extensions/devtools/apidumpmodel.cpp#L38  
This map will need to be kept up to date.

I think it's better like this: 
```
/** APIDOC method
 * Show information message
 * @param {String} title Title
 * @param {String} text Message
 */
void InteractiveApi::info(const QString& title, const QString& text)
```
`info(title, text)` - This part may be constructed, the name should be taken from the method signature, the arguments can be taken either from the signature or from the description. Additional checks can be added if we encounter a mismatch between the description and the signature (but I'm relying more on the review here)